conductor/submitter_maya.py

```diff
@@ -328,6 +345,9 @@ def generateTasksData(self):
 
         # Get logging_verbosity argument for current renderer, e.g. "-ai:lve Debug"  (for arnold).
         verbosity_arg = self.extended_widget.constructVerbosityArg(active_renderer)
+        
+        # For certain renderers, force some flags
+        forced_args = self.extended_widget.getForcedArguments(active_renderer)
```

Comment:
Can you keep the method names consistent? e.g. above we use `construct...Arg` instead of `get...Arguments`.